src/main/webapp/app/exam/manage/exams/exam-update.component.ts

```diff
@@ -330,4 +330,6 @@ export class ExamUpdateComponent implements OnInit {
         this.exam.examUsers = undefined;
         this.exam.studentExams = undefined;
     }
+
+    protected readonly isMessagingEnabled = isMessagingEnabled;
```

Comment:
we usually have these on the top of the file, you already have `readonly FeatureToggle = FeatureToggle;` in this file.